similarity_threshold: 0.7


